ui/cryptomaterial/dropdown.go

```diff
@@ -228,20 +255,55 @@ func (d *DropDown) Layout(gtx C, dropPos int, reversePos bool) D {
 		iRight = dropPos
 	}
 
+	if d.OpenedLayoutInset == nil {
+		d.OpenedLayoutInset = &layout.Inset{
+			Left:  unit.Dp(float32(iLeft)),
+			Right: unit.Dp(float32(iRight)),
+		}
+	}
+
+	if d.Stack {
```

Comment:
Even when this is false, layout.Stack is used (see below). What does this really do?

Reply:
> What does this really do?

It's for a different stack layout as seen in the new dropdown designs(on the dex market page), not referring to the use of the "layout.Stack".